holidayextras/brands
# Changing width of modals

#### What does this PR do? (please provide any background)
This deals with a bug raised in testing for Budget Family Breaks - https://hxshortbreaks.atlassian.net/browse/WEB-10370

#### How can this be tested?
`cd pattern-library && npm link ../brands`

Check that the contact us modal, and the change availability modal aren't going to 90% width.

#### Any tech debt?
No

#### Screenshots / Screencast
![image](https://cloud.githubusercontent.com/assets/18528691/24698423/7c386e4c-19e8-11e7-9d63-fe467c2b40b3.png)

#### What gif best describes how you feel about this work?
![giphy](https://cloud.githubusercontent.com/assets/18528691/24698448/99f2f592-19e8-11e7-8eb4-8df5c3e5266a.gif)


---

**Reviewer**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.
